### PR TITLE
added-erc20-bridging

### PR DESCRIPTION
**Pull request by cedrabner**

### CODE REVIEW

File: docs/use-mainnet/bridges-of-linea/index.mdx [hunk 1]
```diff
@@ -232,3 +232,34 @@ Automatic claiming is only available for ETH L1 to L2 bridging. You will not be
 8. MetaMask will ask you to switch networks and to confirm the transaction from the previous step.
 
 9. You're done! You can check your Ethereum wallet balance on MetaMask or a block explorer such as [etherscan](https://etherscan.io/)!
+
+## How do I bridge ERC20 tokens?
+
+1. Go to the token selection menu.
```
Comment (emilyJLin95) [inline docs/use-mainnet/bridges-of-linea/index.mdx:238]: Can you add the link to where this UI is

File: docs/use-mainnet/bridges-of-linea/index.mdx [hunk 2]
```diff
@@ -232,3 +232,34 @@ Automatic claiming is only available for ETH L1 to L2 bridging. You will not be
...
+
+
+4. Click on the token to select it.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/dogecoin.png")
+      .default
+  }></img>
+
```
Comment (emilyJLin95) [inline docs/use-mainnet/bridges-of-linea/index.mdx:263]: Can you show the before and after states? popup? Is there a video for this too? I find the screenshots unclear

File: docs/use-mainnet/bridges-of-linea/index.mdx (hunk 1 above)
Comment (cedrabner) [reply docs/use-mainnet/bridges-of-linea/index.mdx]: specified that it is the same token bridge. should be good now

File: project-words.txt [hunk 3]
```diff
@@ -758,4 +758,5 @@ PEPE
 LDO
 KNC
 HAPI
-DVF
\ No newline at end of file
+DVF
+dogecoin
```
Comment (emilyJLin95) [inline project-words.txt:762]: Can we keep these alphabetized btw? I guess it doesn't super matter but it's a nit LOL

Comment (cedrabner) [reply project-words.txt:762]: sure lol

Comment (cedrabner) [reply project-words.txt]: done @emilyJLin95

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**